### PR TITLE
#40: o.l.c.m.Throw

### PR DESCRIPTION
**Pull request by dgroup**
#40

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#47](https://github.com/llorllale/cactoos-matchers/issues/47) is now in scope, role is `REV`

File: src/main/java/org/llorllale/cactoos/matchers/Throws.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ * Matcher to check that scalar throw the expected exception.
+ *
+ * <p>Here is an example how {@link Throws} can be used:</p>
+ * <pre>
+ *  MatcherAssert.assertThat(
+ *       "The matcher check that [1,2,3] contains [2]",
+ *       () -> {
+ *            throw new IllegalArgumentException("No object(s) found.");
+ *       },
+ *       new Throw("No object(s) found.", IllegalArgumentException.class)
```
Comment (llorllale) [inline src/main/java/org/llorllale/cactoos/matchers/Throws.java]: @dgroup slight typo there - should be `Throws`

Comment (dgroup) [reply src/main/java/org/llorllale/cactoos/matchers/Throws.java]: @llorllale, sorry, changing...

Comment (dgroup) [reply src/main/java/org/llorllale/cactoos/matchers/Throws.java]: @llorllale done.

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:llorllale/cactoos-matchers#47` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#47](https://github.com/llorllale/cactoos-matchers/issues/47) is now out of scope

**Pull request merged**